dll/win32/msvcrt/stubs.c

```diff
@@ -96,6 +96,7 @@ typedef struct __crt_locale_data_public
                unsigned int _locale_lc_codepage;
 } __crt_locale_data_public;
 
+static
```

Comment:
Should these be just `static`, without `__inline`?

Reply:
Probably. Didn't care enough to check :-)

Reply:
Done